LICENSE

```diff
@@ -1,201 +0,0 @@
-                                 Apache License
```

Comment:
Why is this removed?

Reply:
Addressed in [e53338e](https://github.com/ballerina-platform/module-ballerinax-twilio/pull/209/commits/e53338ef5bd6bd3cc6640678b571a81183fbdbbf)